apps/cli/README.md

```diff
@@ -152,6 +153,28 @@ DESCRIPTION
 
 _See code: [@oclif/plugin-help](https://github.com/oclif/plugin-help/blob/v5.2.9/src/commands/help.ts)_
 
+## `sunodo run`
+
+Run application node.
+
+```
+USAGE
+  $ sunodo run [--blockTime <value>]
+
+FLAGS
+  --blockTime=<value>  [default: 5] interval between blocks in seconds
```

Comment:
I miss the `--verbose` and `--epoch-duration` documentation here.

Reply:
This generated. I'll generate again.